### PR TITLE
[WFLY-15920] IIOP: deprecate legacy security-domain attribute

### PR DESCRIPTION
**Pull request by tadamski**
https://issues.redhat.com/browse/WFLY-15920

### CODE REVIEW

**Comment by tadamski** [comment]
just noticed we have to halt it because of https://github.com/wildfly/wildfly/pull/16107/files

**Comment by tadamski** [comment]
updated

File: iiop-openjdk/src/main/resources/org/wildfly/iiop/openjdk/LocalDescriptions.properties [hunk 1]
```diff
@@ -26,6 +26,7 @@ iiop-openjdk.export-corbaloc=Indicates whether the root context should be export
 # security configuration properties.
 iiop-openjdk.support-ssl=Indicates whether SSL is to be supported (on) or not (off).
 iiop-openjdk.security-domain=The name of the security domain that holds the key and trust stores that will be used to establish SSL connections.
+iiop-openjdk.security-domain.deprecated=The name of the security domain that holds the key and trust stores that will be used to establish SSL connections.
```
Comment (bstansberry) [inline iiop-openjdk/src/main/resources/org/wildfly/iiop/openjdk/LocalDescriptions.properties:29]: This message should be a description of the deprecation, similar to what follows an @deprecated in Javadoc.

Perhaps:

"Legacy security domains are not supported in a running server. Only usable in a domain configuration to support secondary hosts running earlier versions"

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: iiop-openjdk/src/main/resources/org/wildfly/iiop/openjdk/LocalDescriptions.properties (hunk 1 above)
Comment (tadamski) [reply iiop-openjdk/src/main/resources/org/wildfly/iiop/openjdk/LocalDescriptions.properties]: @bstansberry thanks, fixed

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @tadamski

**Pull request merged**